src/content/components/TimelineCanvas.js

```diff
@@ -138,7 +141,9 @@ export default class TimelineCanvas extends Component {
     });
 
     return <canvas className={className}
-                   ref='canvas'
+                   ref={canvas => {
+                     this._canvas = canvas;
+                   }}
```

Comment:
Do you know if the fact that you're passing a lambda here is going to cause unnecessary re-renders? In other places we've used `this._takeCanvasRef` to store the function.
Though it's just really one element, so it probably won't matter.